Content.Server/Administration/Systems/AdminVerbSystem.Smites.cs

```diff
@@ -551,23 +553,35 @@ private void AddSmiteVerbs(GetVerbsEvent<Verb> args)
             };
             args.Verbs.Add(killSign);
 
-            // TODO: Port cluwne outfit.
             Verb clown = new()
             {
-                Text = "Clown",
+                Text = "Cluwne",
                 Category = VerbCategory.Smite,
-                IconTexture = "/Textures/Objects/Fun/bikehorn.rsi/icon.png",
+                IconTexture = "/Textures/Clothing/Mask/cluwne.rsi/icon.png",
                 Act = () =>
                 {
-                    SetOutfitCommand.SetOutfit(args.Target, "ClownGear", EntityManager, (_, clothing) =>
+                    var meta = MetaData(args.Target);
+                    var name = Name(args.Target);
+                    _popupSystem.PopupEntity(Loc.GetString("cluwne-transform", ("target", args.Target)), args.Target, PopupType.LargeCaution);
+
+                    meta.EntityName = Loc.GetString("cluwne-name-prefix", ("target", name));
+
+                    EnsureComp<BackwardsAccentComponent>(args.Target);
+
+                    var vocal = EnsureComp<VocalComponent>(args.Target);
+                    var scream = new SoundCollectionSpecifier ("CluwneScreams");
```

Comment:
Storing things like these in systems a no-no except in very special cases, also this is getting more like a proper implementation of cluwnes than a funny quick admin smite, so having something this reusable behind an admin smite is another big design no-no.
This is why im gonna ask you to move this logic to a cluwne component/system.

Reply:
a new cluwne system and content was created.